<?xml version="1.0" encoding="utf-8"?>
<launch>

  <node pkg="map_generator" 
        type="map_generator_node" 
        name="random_map_sensing" 
        output="screen">
    
    <param name="map/x_size" value="3.0" type="double"/>
    <param name="map/y_size" value="100.0" type="double"/>
    <param name="map/z_size" value="2.0" type="double"/>
    
    <param name="map/resolution" value="0.1" type="double"/>
    <param name="map/map_type" value="fixed" type="string"/>

    <remap from="~obstacle_percentage" to="/map_obstacle_percentage" />
  </node>

  <node pkg="rviz" type="rviz" name="rviz" output="screen" args="-d $(find state_machine)/launch/corridor.rviz"/>
  
</launch>
